### DISCUSSION TITLE
v2/zh/guide/markdown/demo

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#110] [Announcements]
# v2/zh/guide/markdown/demo

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/markdown/demo.html

**Comment by xiaoyu2er** [comment]
react 组件并没有渲染出来

**Reply by Mister-Hope** [reply]
react组件会触发从cdn下载react和react-dom的操作，当两者下载完毕时组件才会被装载。你可以通过选项自行配制这两个cdn链接，如果你担心unpkg在国内访问速度，你可以自行在public文件夹下添加这两个文件并修改cdn链接

**Reply by xiaoyu2er** [reply]
感谢回复！你说的配置文件没有看到在哪里，可以附上链接么，谢谢！

**Reply by Mister-Hope** [reply]
https://vuepress-theme-hope.github.io/v2/zh/config/plugins/md-enhance.html#demo

**Reply by xiaoyu2er** [reply]
非常感谢！

**Comment by xiaoyu2er** [comment]
请问可以选择引入文件么？

**Reply by Mister-Hope** [reply]
那你可以自己引入一个脚本自己mount和展示了。
